Tests/test_content.py

```diff
@@ -841,12 +759,15 @@ def get_json_file(path):
         return json.loads(json_file.read())
 
 
-def execute_testing(tests_settings, server_ip, mockable_tests_names, unmockable_tests_names,
-                    tests_data_keeper, prints_manager, thread_index=0, is_ami=True):
+def execute_testing(tests_settings,
+                    server_ip,
+                    mockable_tests_names,
+                    unmockable_tests_names,
+                    tests_data_keeper,
+                    is_ami=True):
     server = SERVER_URL.format(server_ip)
     server_numeric_version = tests_settings.serverNumericVersion
-    start_message = "Executing tests with the server {} - and the server ip {}".format(server, server_ip)
-    prints_manager.add_print_job(start_message, print, thread_index)
+    logging_manager.info(f'Executing tests with the server {server} - and the server ip {server_ip}', real_time=True)
```

Comment:
Remind me why it is that you set `real_time` in some places? Why specifically here do we want it? This function is sometimes executed in threads.

Reply:
Because these are standalone logs with no context and I think it's better to show them on real time.

I see no damage in only these come up in the middle of a test playbook logs (something that will rarely happen)